### PR TITLE
Admin styles

### PR DESCRIPTION
**Pull request by nadiyafomenko**
Added headline for all pages and move common styles to separate file

### CODE REVIEW

File: src/pages/business-pages/business-page-list.js [hunk 1]
```diff
@@ -27,7 +27,8 @@ const { pathToAddBusinessPage } = config.routes;
 const tableTitles = config.tableHeadRowTitles.businessPages;
 
 const BusinessPageList = () => {
-  const classes = useStyles();
+  const common = useCommonStyles();
```
Comment (moran711) [inline src/pages/business-pages/business-page-list.js:30]: Can you rename this `common` variable to styles or classes?

File: src/pages/categories/categories-page/categories.js [hunk 2]
```diff
@@ -11,13 +11,15 @@ import LoadingBar from '../../../components/loading-bar';
...
-import { useStyles } from './categories.styles';
+import { useCommonStyles } from '../../common';
 import CategoryDeleteDialog from './category-delete-dialog';
+import StandardButton from '../../../components/buttons/standard-button';
 
 const Categories = () => {
   const { IMG_URL } = config;
   const { ADD_CATEGORY } = config.buttonTitles;
 
+  const common = useCommonStyles();
```
Comment (moran711) [inline src/pages/categories/categories-page/categories.js:22]: same

File: src/pages/comments/comments.js [hunk 3]
```diff
@@ -28,7 +27,7 @@ const { REMOVE_COMMENT_MESSAGE, NO_COMMENTS_MESSAGE } = config.messages;
 const { RECENT_COMMENTS } = commentsTranslations;
 
 const Comments = ({ productId }) => {
-  const classes = useStyles();
+  const common = useCommonStyles();
```
Comment (moran711) [inline src/pages/comments/comments.js:30]: same

File: src/pages/common.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
+import { makeStyles } from '@material-ui/core/styles';
```
Comment (moran711) [inline src/pages/common.js:1]: Can you rename this file to `common.styles.js` due to our style guide.

File: src/pages/contacts-page/contacts-page.js [hunk 5]
```diff
@@ -29,8 +30,10 @@ const pathToAddContactPage = config.routes.pathToAddContact;
 const tableTitles = config.tableHeadRowTitles.contacts;
 
 const ContactsPage = () => {
+  const common = useCommonStyles();
```
Comment (moran711) [inline src/pages/contacts-page/contacts-page.js:33]: Can you rename this `common` variable to styles or classes?

File: src/pages/email-questions/email-questions-list.js [hunk 6]
```diff
@@ -30,6 +31,8 @@ const tableTitles = tableHeadRowTitles.emailQuestions;
 
 const EmailQuestionsList = () => {
   const styles = useStyles();
+  const common = useCommonStyles();
```
Comment (moran711) [inline src/pages/email-questions/email-questions-list.js:34]: Can you rename this `common` variable to styles or classes?

File: src/pages/header/header-page/header-page.js [hunk 7]
```diff
@@ -22,7 +22,8 @@ const pathToHeaderAddPage = routes.pathToAddHeader;
 const tableTitles = config.tableHeadRowTitles.headers;
 
 const HeaderPage = () => {
-  const styles = useStyles();
+  const common = useCommonStyles();
```
Comment (moran711) [inline src/pages/header/header-page/header-page.js:25]: Can you rename this `common` variable to styles or classes?

File: src/pages/home-page-slides/slides-order/slides-order.js [hunk 8]
```diff
@@ -14,10 +14,14 @@ import {
   setSlides,
   updateSlidesOrder
 } from '../../../redux/home-page-slides/home-page-slides.actions';
+import { useCommonStyles } from '../../common';
 
 const SlidesOrder = (props) => {
   const styles = useStyles();
+  const common = useCommonStyles();
```
Comment (moran711) [inline src/pages/home-page-slides/slides-order/slides-order.js:21]: Can you rename this `common` variable to styles or classes?

File: src/pages/home-page-slides/slides-page/slides-page.js [hunk 9]
```diff
@@ -27,9 +28,13 @@ const { mainPageTitle, slideTitle } = config.titles.homePageSliderTitle;
 const { pathToAddHomePageSlide } = config.routes;
 
 const SlidesPage = () => {
-  const { openSuccessSnackbar } = useSuccessSnackbar();
   const styles = useStyles();
+  const common = useCommonStyles();
```
Comment (moran711) [inline src/pages/home-page-slides/slides-page/slides-page.js:32]: Can you rename this `common` variable to styles or classes?

File: src/pages/home-page/home-page.js [hunk 10]
```diff
@@ -14,13 +14,17 @@ import {
...
 import { useStyles } from './home-page.styles';
+import { useCommonStyles } from '../common';
 
-const { homePageEdit } = titles;
+const { homePageTitles } = titles;
 const { IMG_URL } = config;
 
 const HomePage = () => {
+  const styles = useStyles();
+  const common = useCommonStyles();
```
Comment (moran711) [inline src/pages/home-page/home-page.js:24]: Can you rename this `common` variable to styles or classes?

File: src/pages/material/material-page/material-page.js [hunk 11]
```diff
@@ -27,6 +28,8 @@ const tableTitles = config.tableHeadRowTitles.materials;
 
 const MaterialPage = () => {
   const styles = useStyles();
+  const common = useCommonStyles();
```
Comment (moran711) [inline src/pages/material/material-page/material-page.js:31]: Can you rename this `common` variable to styles or classes?

File: src/pages/model/model-page/model-page.js [hunk 12]
```diff
@@ -28,6 +29,8 @@ const tableTitles = config.tableHeadRowTitles.models;
 
 const ModelPage = () => {
   const styles = useStyles();
+  const common = useCommonStyles();
```
Comment (moran711) [inline src/pages/model/model-page/model-page.js:32]: Can you rename this `common` variable to styles or classes?

File: src/pages/news/news-page/news-page.js [hunk 13]
```diff
@@ -25,7 +26,9 @@ const pathToNewsAddPage = config.routes.pathToAddNews;
 const tableTitles = config.tableHeadRowTitles.news;
 
 const NewsPage = () => {
-  const classes = useStyles();
+  const styles = useStyles();
+  const common = useCommonStyles();
```
Comment (moran711) [inline src/pages/news/news-page/news-page.js:30]: Can you rename this `common` variable to styles or classes?

File: src/pages/orders/orders-page/orders-page.js [hunk 14]
```diff
@@ -17,6 +18,7 @@ import { config } from '../../../configs';
 
 const OrdersPage = () => {
   const styles = useStyles();
+  const common = useCommonStyles();
```
Comment (moran711) [inline src/pages/orders/orders-page/orders-page.js:21]: Can you rename this `common` variable to styles or classes?

File: src/pages/pattern/pattern-page/pattern-page.js [hunk 15]
```diff
@@ -26,7 +27,10 @@ const tableTitles = config.tableHeadRowTitles.patterns;
 
 const PatternPage = () => {
   const styles = useStyles();
+  const common = useCommonStyles();
```
Comment (moran711) [inline src/pages/pattern/pattern-page/pattern-page.js:30]: Can you rename this `common` variable to styles or classes?

File: src/pages/products/products-page/products-nav/products-nav.js [hunk 16]
```diff
@@ -1,42 +1,38 @@
...
 import { productsTranslations } from '../../../../translations/product.translations';
 import routes from '../../../../configs/routes';
+import { useCommonStyles } from '../../../common';
+import { config } from '../../../../configs';
 
 const { ADD_PRODUCT } = productsTranslations;
 
 const ProductsNav = () => {
-  const styles = useStyles();
+  const common = useCommonStyles();
```
Comment (moran711) [inline src/pages/products/products-page/products-nav/products-nav.js:16]: same

File: src/pages/products/products-page/products-page.js [hunk 17]
```diff
@@ -31,7 +31,8 @@ const tableTitles = config.tableHeadRowTitles.products;
 const { imagePrefix } = config;
 
 const ProductsPage = () => {
-  const styles = useStyles();
+  const common = useCommonStyles();
```
Comment (moran711) [inline src/pages/products/products-page/products-page.js:34]: same

File: src/pages/statistic/statistic-page.js [hunk 18]
```diff
@@ -28,7 +29,9 @@ const {
 } = config;
 
 const StatisticPage = () => {
+  const common = useCommonStyles();
```
Comment (moran711) [inline src/pages/statistic/statistic-page.js:32]: same

File: src/pages/users/users-page/users-page.js [hunk 19]
```diff
@@ -17,6 +18,8 @@ const userTabNames = config.tabNames.users;
 
 const UsersPage = () => {
   const styles = useStyles();
+  const common = useCommonStyles();
```
Comment (moran711) [inline src/pages/users/users-page/users-page.js:21]: same

**Review by vasylshpak — APPROVED** [review]

File: src/pages/business-pages/business-page-list.js (hunk 1 above)
Comment (nadiyafomenko) [reply src/pages/business-pages/business-page-list.js]: fixed

File: src/pages/categories/categories-page/categories.js (hunk 2 above)
Comment (nadiyafomenko) [reply src/pages/categories/categories-page/categories.js]: fixed

File: src/pages/comments/comments.js (hunk 3 above)
Comment (nadiyafomenko) [reply src/pages/comments/comments.js]: fixed

File: src/pages/common.js (hunk 4 above)
Comment (nadiyafomenko) [reply src/pages/common.js]: fixed

File: src/pages/contacts-page/contacts-page.js (hunk 5 above)
Comment (nadiyafomenko) [reply src/pages/contacts-page/contacts-page.js]: fixed

File: src/pages/email-questions/email-questions-list.js (hunk 6 above)
Comment (nadiyafomenko) [reply src/pages/email-questions/email-questions-list.js]: fixed

File: src/pages/header/header-page/header-page.js (hunk 7 above)
Comment (nadiyafomenko) [reply src/pages/header/header-page/header-page.js]: fixed

File: src/pages/home-page-slides/slides-order/slides-order.js (hunk 8 above)
Comment (nadiyafomenko) [reply src/pages/home-page-slides/slides-order/slides-order.js]: fixed

File: src/pages/home-page-slides/slides-page/slides-page.js (hunk 9 above)
Comment (nadiyafomenko) [reply src/pages/home-page-slides/slides-page/slides-page.js]: fixed

File: src/pages/home-page/home-page.js (hunk 10 above)
Comment (nadiyafomenko) [reply src/pages/home-page/home-page.js]: fixed

File: src/pages/material/material-page/material-page.js (hunk 11 above)
Comment (nadiyafomenko) [reply src/pages/material/material-page/material-page.js]: fixed

File: src/pages/model/model-page/model-page.js (hunk 12 above)
Comment (nadiyafomenko) [reply src/pages/model/model-page/model-page.js]: fixed

File: src/pages/news/news-page/news-page.js (hunk 13 above)
Comment (nadiyafomenko) [reply src/pages/news/news-page/news-page.js]: fixed

File: src/pages/orders/orders-page/orders-page.js (hunk 14 above)
Comment (nadiyafomenko) [reply src/pages/orders/orders-page/orders-page.js]: fixed

File: src/pages/pattern/pattern-page/pattern-page.js (hunk 15 above)
Comment (nadiyafomenko) [reply src/pages/pattern/pattern-page/pattern-page.js:30]: fixed

File: src/pages/products/products-page/products-nav/products-nav.js (hunk 16 above)
Comment (nadiyafomenko) [reply src/pages/products/products-page/products-nav/products-nav.js:16]: fixed

File: src/pages/products/products-page/products-page.js (hunk 17 above)
Comment (nadiyafomenko) [reply src/pages/products/products-page/products-page.js:34]: fixed

File: src/pages/users/users-page/users-page.js (hunk 19 above)
Comment (nadiyafomenko) [reply src/pages/users/users-page/users-page.js:21]: fixed

File: src/pages/statistic/statistic-page.js (hunk 18 above)
Comment (nadiyafomenko) [reply src/pages/statistic/statistic-page.js:32]: fixed

**Review by moran711 — APPROVED** [review]

**Review by VitalikVoicix — APPROVED** [review]

**Comment by vasylshpak** [comment]
Great job! Please check the conflicts, and we are ready to merge your PR  🚀

**Pull request merged**